src/node_i18n.cc

```diff
@@ -51,17 +59,43 @@ extern "C" const char U_DATA_API SMALL_ICUDATA_ENTRY_POINT[];
 
 namespace node {
 
+using v8::ArrayBuffer;
 using v8::Context;
 using v8::FunctionCallbackInfo;
 using v8::Local;
 using v8::Object;
 using v8::String;
+using v8::Uint32;
+using v8::Uint8Array;
 using v8::Value;
 
 bool flag_icu_data_dir = false;
 
 namespace i18n {
 
+#define THROW_AND_RETURN_UNLESS_BUFFER(env, obj)                              \
```

Comment:
How about moving these macros into `util.h`? I think I would prefer that over copypasting.